src/main/resources/story/web/login.story

```diff
@@ -0,0 +1,27 @@
+Scenario: Log-in
```

Comment:
Please create a composite step with positive login. This functionality will be used in scope of other tests.